server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -221,6 +221,7 @@ private ConfigProperties() {
 
     public static final String STANDALONE = "candlepin.standalone";
     public static final String ENV_CONTENT_FILTERING = "candlepin.environment_content_filtering";
+    public static final String USE_SYSTEM_UUID = "candlepin.use_system_uuid";
```

Comment:
I can't help but think that this property isn't descriptive enough for what it does, but can't think of an alternative that would be better, so I suppose it's fine as is.